ref/javadocs/.gitignore

```diff
@@ -0,0 +1 @@
+!*.html
```

Comment:
I did this on purpose. The top level .gitignore hides *.html from git which is good for most content since it means if you convert adoc to html you wont' accidentally checkin the adoc. However it means we can't checkin new javadoc because it is all html. I found this allowed me to create html for javadoc.